backend/hitas/views/postal_code.py

```diff
@@ -1,7 +1,15 @@
 from rest_framework import serializers
 
 from hitas.models import HitasPostalCode
-from hitas.views.utils import HitasModelSerializer, HitasModelViewSet
+from hitas.views.utils import HitasCharFilter, HitasFilterSet, HitasModelSerializer, HitasModelViewSet
+
+
+class HitasPostalCodeFilterSet(HitasFilterSet):
+    value = HitasCharFilter(lookup_expr="contains")
```

Comment:
Use case here is to enable filtering for postal codes. Before this change PostalCodes could not be filtered as the view did not have a FilterSet, because they don't inherit from `AbstractCodeViewSet` anymore, so the filter set is not autocreated behind the scenes.

In frontend we use a selector for the postal codes, so they need to be filterable.

Reply:
I was more thinking about the `contains` search instead of `exact`